docs/content/index.md

```diff
@@ -1,11 +1,16 @@
-# ProjectScaffold
+##What is ProjectScaffold?
+ProjectScaffold helps you get started with a new F# .NET/Mono project solution with everything needed for successful organising of code, tools and publishing. 
```

Comment:
They are already working. It's just not created during init yet.
On Apr 30, 2015 12:20 PM, "Indy Garcia" <notifications@github.com> wrote:

> In docs/content/index.md
> <https://github.com/fsprojects/ProjectScaffold/pull/175#discussion_r29418074>
> :
>
> > @@ -1,11 +1,16 @@
> > -# ProjectScaffold
> > +##What is ProjectScaffold?
> > +ProjectScaffold helps you get started with a new F# .NET/Mono project solution with everything needed for successful organising of code, tools and publishing.
>
> Sure, but wouldn't it be clearer if we have F# now and other langs added
> when they become available.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fsprojects/ProjectScaffold/pull/175/files#r29418074>.
>